docs/concepts/blocks.md

```diff
@@ -13,11 +13,113 @@ tags:
 
 Blocks are a primitive within Prefect that enable the storage of configuration and provide an interface for interacting with external systems.
 
-Blocks are useful for configuration that needs to be shared across flow runs and between flows. For configuration that will change between flow runs, we recommend using [parameters](/concepts/flows/#parameters).
-
-With blocks, you are able to securely store credentials for authenticating with services like AWS, GitHub, Slack, or any other system you'd like to orchestrate with Prefect. Blocks also expose methods that provide pre-built functionality for performing actions against an external system. Blocks can be used to download data from or upload data to an S3 bucket, query data from or write data to a database, or send a message to a Slack channel.
-
-Blocks can also be created by anyone and shared with the community. You'll find blocks that are available for consumption in many of the published [Prefect Collections](/collections/catalog).
+With blocks, you are able to securely store credentials for authenticating with services like AWS, GitHub, Slack, or any other system you'd like to orchestrate with Prefect. 
+
+Blocks also expose methods that provide pre-built functionality for performing actions against an external system. Blocks can be used to download data from or upload data to an S3 bucket, query data from or write data to a database, or send a message to a Slack channel.
+
+You may configure blocks through code or via the Prefect Cloud and the Prefect server UI.
+
+You can access blocks for both configuring flow [deployments](/concepts/deployments/) and directly from within your flow code.
+
+Prefect provides some built-in block types that you can use right out of the box. Additional blocks are available through [Prefect Collections](/collections/catalog/) &mdash to use these blocks you can `pip install` the collection, then register the blocks you want to use with Prefect Cloud or a Prefect server.
+
+Prefect Cloud and the Prefect server UI display a library of block types available for you to configure blocks that may be used by your flows.
+
+![Viewing the new block library in the Prefect UI](../img/ui/block-library.png)
+
+!!! tip "Blocks and parameters"
+    Blocks are useful for configuration that needs to be shared across flow runs and between flows.
+
+    For configuration that will change between flow runs, we recommend using [parameters](/concepts/flows/#parameters).
+
+## Prefect built-in blocks
```

Comment:
Are we pulling these from somewhere or will this be copy & paste to maintain?

Reply:
At the moment this is semi-automated, but does require some hand work. Keeping an eye on making it fully automated in future.

Reply:
To be more specific, the majority of the table is pulled directly from JSON in our repos, but the JSON does not yet include some details such as documentation URLs for links. We'll work on filling that our over time to enable build-time automation for the entire table.

Reply:
The integrations team can work on making sure that all blocks have a documentation URL and resolve any other missing metadata. If this table would benefit from a custom view for blocks (like with relative URLs for the blocks that are in the core library) let us know and we can work on building that.

Reply:
@desertaxle documentation URLs would be great. I mentioned to Evan and Nate including something like a collection_url as well, but I'm inferring this from the documentation URL in my script, so not critical.